docs/source/upcoming_release_notes/1201-modify_sp1k4-att_RTD_PV_to_match_the_prefix_SPEC.rst

```diff
@@ -0,0 +1,30 @@
+1201 modify sp1k4-att RTD PV to match the prefix SPEC
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- Add SPEC into SP1K4 Attenuator RTD to match prefix for SP1K4 group device
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- @tongju
```

Comment:
```suggestion
- tongju12
```

This is your github username

Reply:
Thanks Robert. I have not tested yet because the PLC code is still under review. Zach asked me to modify the this in the comments. I need to work more on PLC code and push, then rebuild IOC to test. I will upload the test here. Please leave it until I post test result here.

Reply:
I tested today after I rebuilt IOC: find the two new PVs:[
tongju@tmo-control ~]$ caget SP1K4:SPEC:ATT:RTD:01:TEMP_RBV
SP1K4:SPEC:ATT:RTD:01:TEMP_RBV 19.56
[tongju@tmo-control ~]$ caget SP1K4:SPEC:ATT:RTD:02:TEMP_RBV
SP1K4:SPEC:ATT:RTD:02:TEMP_RBV 19.41

Reply:
Also change the doc from @tongju to @tongju12 as suggested